### PR TITLE
:book: Instructions for deploying KS v0.21 using K3D

### PR DESCRIPTION
**Pull request by eaepstein**
## Summary
KS install instructions tested on ubuntu 22.04 using K3D

### CODE REVIEW

**Comment by ezrasilvera** [comment]
@eaepstein You need to sign of your commits (see https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).
You can `git amend`  old commit to fix it

File: docs/content/direct/deploy-on-k3d.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+# Deploy KubeStellar on K3D
+
+This document shows how to deploy kubestellar v0.21.0 and a managed cluster on K3D
```
Comment (ezrasilvera) [inline docs/content/direct/deploy-on-k3d.md:3]: Maybe drop the eversion?  
Maybe: ".. How to use K3D as the clusters for the hub and/or WECs for KubeStellar "

File: docs/content/direct/deploy-on-k3d.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+## Prereqs
+
+In addition to [pre-reqs](pre-reqs.md), install k3d v5.6.0 (only k3d version tested so far)
+
+## Common Setup for standard examples
+
+1. You may want to `set -e` in your shell so that any failures in the setup or usage scenarios are not lost.
+
+1. If you previously installed KS on K3D:
```
Comment (ezrasilvera) [inline docs/content/direct/deploy-on-k3d.md:13]: I don't think we need this section.  If you previously installed KS wit k3d , you don't use this guide :-)

File: docs/content/direct/deploy-on-k3d.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    ```
+
+1. Create a K3D hosting cluster with nginx ingress controller:
+    ```shell
+    k3d cluster create -p "9443:443@loadbalancer" --k3s-arg "--disable=traefik@server:*" kubeflex
+    helm install ingress-nginx ingress-nginx --repo https://kubernetes.github.io/ingress-nginx --version 4.6.1 --namespace ingress-nginx --create-namespace
+    kubectl config rename-context k3d-kubeflex kubeflex
+    ```
+
+1. [Paolo says] when we use kind, the name of the container is kubeflex-control-plane and that is what we use
```
Comment (ezrasilvera) [inline docs/content/direct/deploy-on-k3d.md:35]: [Paolo..] This should be remove on public doc ..

File: docs/content/direct/deploy-on-k3d.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+    docker start kubeflex-control-plane
+    ```
+    Wait 1-2 minutes for all pods to be restarted.
+    Use the following command to confirm all are fully running:
+    ```shell
+    kubectl --context kubeflex get po -A
+    ```
+
+1. Install kubestellar controller and OCM space:
+   [Paolo says] we are using nginx ingress with tls passthru.
```
Comment (ezrasilvera) [inline docs/content/direct/deploy-on-k3d.md:50]: Same

File: docs/content/direct/deploy-on-k3d.md (hunk 1 above)
Comment (eaepstein) [reply docs/content/direct/deploy-on-k3d.md:3]: Thanks, modified

File: docs/content/direct/deploy-on-k3d.md (hunk 3 above)
Comment (eaepstein) [reply docs/content/direct/deploy-on-k3d.md:35]: Done

File: docs/content/direct/deploy-on-k3d.md (hunk 4 above)
Comment (eaepstein) [reply docs/content/direct/deploy-on-k3d.md:50]: Done

**Pull request closed without merging**